proto/gen.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+# see: https://stackoverflow.com/a/246128
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+
+cd $SCRIPT_DIR
+protoc -I=. -I=$GOPATH/src -I=$GOPATH/src/github.com/gogo/protobuf --gofast_out=../types/pb/optimint optimint.proto
```

Comment:
I think the tendermint team uses a docker image for this. To help other engineers to collaborate with you, it might be a good idea to use something like that too.
Otherwise (at least from my experience) there will be lots of questions: which protobuf version, which compiler exactly, how do I build it etc?

Reply:
I saw that Docker approach in Tendermint. Personally I'm not a big fan of dockerising the simplest tasks. But as you said, it may be very beneficial in this particular case. I think it's not the highest priority right now, so I created followup issue: https://github.com/lazyledger/optimint/issues/75.

Reply:
Thanks